qkeras/__init__.py

```diff
@@ -36,4 +36,4 @@
 
 assert tf.executing_eagerly(), "QKeras requires TF with eager execution mode on"
 
-__version__ = "0.9.0"
+__version__ = "0.10.0"
```

Comment:
It looks like you are making a new QKeras version? If so, we need to tie it to a specific release QKeras 0.10.0

Perhaps it is better to hold off on the version update until the PR is submitted, then we create a separate change with the version / release update?

Reply:
This was requested here: https://github.com/google/qkeras/pull/118#issuecomment-1530391917